models/restful-model-collection.coffee

```diff
@@ -63,6 +63,19 @@ class RestfulModelCollection
       callback(err) if callback
       Promise.reject(err)
 
+  expand: (id, callback = null) ->
```

Comment:
Expanded view should be available for models including Message, as the example in the platform docs recommends using view: expanded for them: https://nylas.com/docs/platform/?node#expanded_message_view